README.win32.adoc

```diff
@@ -100,22 +100,18 @@ The native-code compiler (`ocamlopt`) and static linking of OCaml bytecode with
 C code (`ocamlc -custom`) require a Microsoft Visual C/C++ Compiler and the
 `flexlink` tool (see <<bmflex,above>>).
 
-Any edition (including Express/Community editions) of Microsoft Visual Studio
-2008 or later may be used to provide the required Windows headers and the C
-compiler. Additionally, some older Microsoft Windows SDKs include the
-Visual C/C++ Compiler as well as the Build Tools for Visual Studio.
+The OCaml runtime requires C11 atomics, which are available in
+Visual Studio 2022 (starting from release 17.8). Earlier versions (either of
+Visual Studio 2022 17.7 and earlier or Visual Studio 2019 and earlier) cannot
+be used to compile OCaml. In addition to the paid versions of Visual Studio,
+OCaml can be compiled with either Visual Studio 2022 Community or Build Tools
+for Visual Studio 2022. The following components are needed (the precise version
+numbers may differ, but this should not matter):
+
+ - MSVC v143 - VS 2022 C++ x64/x86 build tools (Latest)
+ - C\++/CLI support for v143 build tools (Latest)
```

Comment:
```suggestion
 - C++/CLI support for v143 build tools (Latest)
```

I suppose this was copied from asciidoc.

Reply:
It is asciidoc? But it also wasn't rendering correctly - fixed, thanks!